jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     "^.+\\.(ts|js)$": "<rootDir>/node_modules/babel-jest"
   },
   "transformIgnorePatterns": [
-    "node_modules/(?!(ol|@terrestris/*)/)"
+    "node_modules/(?!(ol|geostyler-style|@terrestris/*)/)"
```

Comment:
I don't think this is needed anymore, as `geostyler-style` is no transpiled to `*.js` and `*.d.ts`.

Reply:
Right. Fixed.